src/chapters/setup-intro-data-types/exercises.rst

```diff
@@ -1,2 +1,210 @@
 Exercises: Data Types
 ======================
+
+Initial Setup
+-------------
+
+The steps here will walk you through setting up a repository that you’ll
+use to study example code, work on studios, and complete your first
+assignment of this unit.
+
+#. Create a *Fork* of `LaunchCodeEducation/java-web-dev-exercises <https://github.com/LaunchCodeEducation/java-web-dev-exercises>`__.
+   Do not clone and create a local version of this repo just yet. 
+   
+#. Open IntelliJ.
+
+   .. note::
+
+      If the app opens up to an existing project, select *IntelliJ >
+      Preferences > Appearance & Behavior > System Settings* and uncheck
+      *Reopen last project on startup*. (For Windows users: *File >
+      Settings > Appearance & Behavior > System Settings*.) Close and
+      Reopen IntelliJ.
+
+#. From the “Welcome to IntelliJ” dialog, select *Check out from Version
+   Control > Git*
+   **TODO: Get screenshots of this process.**
+#. Click the button on the lower left corner of the dialog to configure your Github account.
+#. To work with a remote repository in IntelliJ, you need to configure the program to access 
+   your GitHub account. We recommend taking the extra step of authenticating using a token. 
+   This takes only one brief extra step, and will prevent you from having to update IntelliJ
```

Comment:
"extra step" is repeated over two sentences.  We can probably rephrase to remove one of the comments.